src/publicatie_tabellen/db_functions/README.md

```diff
@@ -0,0 +1,22 @@
+## When the function code is changed
+
+Unmigrate the `0002_db_functions migration` in the app `publicatie_tabellen` and then migrate again.
```

Comment:
you just need to go back in the migrations because the code connection of the sql-function is changed. All migrations need to migrated again, with no changes in de migration.